code/api/tests/user.test.js

```diff
@@ -38,7 +38,7 @@ describe("user queries", () => {
     expect(response.body.data.user.role).toEqual('ADMIN')
     expect(response.body.data.user.id).toEqual(1)
     expect(response.body.data.user.email).toEqual('admin@crate.com')
-    expect(response.body.data.user.image).toEqual('https://images.unsplash.com/photo-1537815749002-de6a533c64db?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1145&q=80')
+    expect(response.body.data.user.image).toEqual('https://images.unsplash.com/photo-1537151608828-ea2b11777ee8?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=939&q=80')
```

Comment:
Updated this back to the default image, it should now pass for everyone

Reply:
Perfect, thanks for taking a look at that!